src/databricks/labs/ucx/source_code/notebooks/base.py

```diff
@@ -0,0 +1 @@
+NOTEBOOK_HEADER = "Databricks notebook source"
```

Comment:
perhaps the `cells.py` was not the best name for the purpose. `nbformat.py` night be a better name. could you bring it back to cells.py?

Reply:
the file name isn't the problem here. This is about avoiding cyclic imports.